data/hai/hai missions.txt

```diff
@@ -157,6 +243,17 @@ mission "First Contact: Unfettered"
 
 
 
+mission "First Contact: Unfettered"
```

Comment:
This mission will become unnecessary when using a branch, but in the future know that you shouldn't create dummy missions that do absolutely nothing but fail. Instead, you would do the following for the previous missions:
```
on decline
	set "First Contact: Unfettered: offered"
```

This avoids the need of a dummy missions.

Reply:
That was one of the first things I tried, but it failed. I'm sure I did something wrong, though.